feat: adding the OOP platformer project to new JS curriculum (#50385)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64ab0134716d0a7c8889f167.md

---
id: 64ab0134716d0a7c8889f167
title: Step 34
challengeType: 0
dashedName: step-34
---

# --description--

> In order to see the player on the screen, it will need to be drawn on the screen.  

Reviewer: ```suggestion
To visualize the player on the screen, it needs to be drawn on the canvas.
```
